### PR TITLE
L5 Implement RPC methods for licensing

### PR DESCRIPTION
**Pull request by PJLewandowski**

### CODE REVIEW

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -2319,6 +2319,41 @@ UniValue scantxoutset(const JSONRPCRequest& request)
...
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw std::runtime_error(
+            RPCHelpMan{"getlicensedminers",
+                "\nReturns the list of licensed miners.\n",
+                {},
+				RPCResult{
+				            "\"licenses\": [\n"
+				            "    {\n"
+				            "      \"height\": n,                  (numeric) Height of the last modifiaction of given license\n"
```
Comment (mtwaro) [inline src/rpc/blockchain.cpp]: spelling

File: src/rpc/blockchain.cpp [hunk 2]
```diff
@@ -2319,6 +2319,41 @@ UniValue scantxoutset(const JSONRPCRequest& request)
...
+				            "      \"hashrate\": n,                (numeric) Hashrate assigned to miner where 1 assigned unit is 10 PH\n"
```
Comment (mtwaro) [inline src/rpc/blockchain.cpp]: I suggest to name the variable as a self-explanatory `hashrate_ph` and show real hashrate in PH

File: src/rpc/blockchain.cpp (hunk 1 above)
Comment (mtwaro) [reply src/rpc/blockchain.cpp]: It should be `Height of the last modification of a given license`

File: src/rpc/blockchain.cpp (hunk 2 above)
Comment (mtwaro) [reply src/rpc/blockchain.cpp]: `Hashrate assigned to the miner`

File: src/rpc/blockchain.cpp (hunk 1 above)
Comment (PJLewandowski) [reply src/rpc/blockchain.cpp]: Done

File: src/rpc/blockchain.cpp (hunk 2 above)
Comment (PJLewandowski) [reply src/rpc/blockchain.cpp]: Done

File: src/rpc/blockchain.cpp [hunk 3]
```diff
@@ -2329,8 +2330,8 @@ static UniValue getlicensedminers(const JSONRPCRequest& request)
 				RPCResult{
 				            "\"licenses\": [\n"
 				            "    {\n"
-				            "      \"height\": n,                  (numeric) Height of the last modifiaction of given license\n"
-				            "      \"hashrate\": n,                (numeric) Hashrate assigned to miner where 1 assigned unit is 10 PH\n"
+				            "      \"height\": n,                  (numeric) Height of the last modification of a given license\n"
+				            "      \"hashrate_ph\": n,             (numeric) Hashrate assigned to miner\n"
```
Comment (mtwaro) [inline src/rpc/blockchain.cpp]: Hashrate assigned to miner in PH/s

File: src/rpc/blockchain.cpp [hunk 4]
```diff
@@ -2346,7 +2347,7 @@ static UniValue getlicensedminers(const JSONRPCRequest& request)
     for (const auto& license : minerLicenses.GetLicenses()) {
     	UniValue licenseObj(UniValue::VOBJ);
     	licenseObj.pushKV("height", license.height);
-    	licenseObj.pushKV("hashrate", license.hashRate);
+    	licenseObj.pushKV("hashrate", license.hashRate * 10);
```
Comment (mtwaro) [inline src/rpc/blockchain.cpp]: The minimum assigned unit should be 1 PH, not 10

Comment (PJLewandowski) [reply src/rpc/blockchain.cpp]: Done

**Review by mtwaro — APPROVED** [review]

**Pull request merged**